etheno/echidna.py

```diff
@@ -196,4 +196,4 @@ def emit_transaction(self, txn):
 
 
 if __name__ == '__main__':
-    install_echidna(allow_reinstall=True)
```

Comment:
this file can probably be removed if the echidna integration is no longer there 🤔